### ISSUE TITLE
Grid loader doesn't display when scrollable is 'non'

### ISSUE THREAD

**Issue by leniency** [telerik/kendo-vue#256]
Adding `:scollable="'none'"` causes the grid load to no longer display with the native grid:

```vue
<Grid
      :loader="true"
      :scrollable="'none'"
      :data-items="result"
      :columns="columns"
    >
    </Grid>
```

https://codesandbox.io/s/cranky-wood-ohkvij

**Comment by zdravkov** [comment]
Thank you for reporting the issue, @leniency. We have successfully replicated it and will do our best to fix it as soon as possible.
As a side note - while we do monitor the issues submitted in this repository, sometimes we may miss some of them or accidentally delay answers for them. This is because in Kendo UI for Vue we have dedicated support channels for direct support questions, forums and feedback portal where we are able to provide the best support possible. So in case you have more questions we recommend using any of them - https://www.telerik.com/support/kendo-ui-for-vue.
Greetings,
Plamen
